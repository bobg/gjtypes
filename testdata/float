6.283185307
