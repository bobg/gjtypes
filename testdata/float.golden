var data float64 // Unmarshal into this type.

type S001 struct {
}
